movieland-persistence-jdbc/src/main/java/com/gromoks/movieland/dao/JdbcGenreDao.java

```diff
@@ -17,12 +24,20 @@
 
     private final GenreRowMapper genreRowMapper = new GenreRowMapper();
 
+    private final MovieToGenreRowMapper movieToGenreRowMapper = new MovieToGenreRowMapper();
+
     @Autowired
     private JdbcTemplate jdbcTemplate;
```

Comment:
you don't need to have both: jdbcTemplate and named.

Reply:
I use it for getAll